src/OpenSpaceToolkit/Physics/Coordinate/Frame/Providers/IERS/Manager.cpp

```diff
@@ -82,55 +82,35 @@ Directory Manager::getFinals2000ADirectory() const
     return Directory::Path(localRepository_.getPath() + Path::Parse("finals-2000A"));
 }
 
-Array<BulletinA> Manager::getBulletinAArray() const
+BulletinA Manager::getBulletinA()
 {
-    return aBulletins_;
-}
-
-BulletinA Manager::getBulletinAAt(const Instant& anInstant) const
-{
-    if (!anInstant.isDefined())
-    {
-        throw ostk::core::error::runtime::Undefined("Instant");
-    }
-
     std::lock_guard<std::mutex> lock {mutex_};
 
-    const BulletinA* bulletinAPtr = this->accessBulletinAAt(anInstant);
+    const BulletinA* bulletinAPtr = this->accessBulletinA();
 
     if (bulletinAPtr != nullptr)
     {
         return *bulletinAPtr;
     }
 
-    throw ostk::core::error::RuntimeError("Cannot obtain Bulletin A at [{}].", anInstant.toString());
-}
-
-Array<Finals2000A> Manager::getFinals2000AArray() const
-{
-    return finals2000aArray_;
+    throw ostk::core::error::RuntimeError("Cannot obtain Bulletin A.");
 }
 
-Finals2000A Manager::getFinals2000AAt(const Instant& anInstant) const
+Finals2000A Manager::getFinals2000A()
 {
-    if (!anInstant.isDefined())
-    {
-        throw ostk::core::error::runtime::Undefined("Instant");
-    }
-
     std::lock_guard<std::mutex> lock {mutex_};
 
-    const Finals2000A* finals2000aPtr = this->accessFinals2000AAt(anInstant);
+    const Finals2000A* finals2000aPtr = this->accessFinals2000A();
 
     if (finals2000aPtr != nullptr)
     {
         return *finals2000aPtr;
     }
```

Comment:
Should these be made Unique pointers (or shared) instead of raw pointers? These can still be compared to `nullptr` if I remember correctly

Reply:
note to self, can be fixed in future ^